src/app/messages.service.ts

```diff
@@ -23,10 +24,9 @@ export class MessagesService {
   constructor(private httpClient: HttpClient) { }
 
   getMessages(id:string|null):Observable<Message[]> {
-    return timer(1, 3000)
+    return timer(1, 500)
     .pipe(
-      tap(x => console.log("hi", x)),
-      switchMap(x => this.httpClient.get<Message[]>
+        switchMap(x => this.httpClient.get<Message[]>
         (`http://localhost:8080/api/conversations/${id}/messages`)),
```

Comment:
Please replace with a reference to a property in the environment files

Reply:
done.